### ISSUE TITLE
(TESTNET) Only 6 USDT symbols found

### ISSUE THREAD

**Issue by Amraki** [CyberPunkMetalHead/Binance-volatility-trading-bot#18]
To be clear, this is based on TESTNET data.

The following is a list of all symbols returned from `client.get_all_tickers()`

```
BNBBUSD, BTCBUSD, ETHBUSD, LTCBUSD, TRXBUSD, XRPBUSD
BNBUSDT, BTCUSDT, ETHUSDT, LTCUSDT, TRXUSDT, XRPUSDT
BNBBTC, ETHBTC, LTCBTC, TRXBTC, XRPBTC
LTCBNB, TRXBNB, XRPBNB
```

Should there be so few symbols available to test with? 
If so, this makes it difficult to test changes as I primarily see "No coins moved more than 3% in the last 5 minute(s)".

**Comment by CyberPunkMetalHead** [comment]
Yep, that's pretty much all we can get with the testnet, which indeed makes this tricky to test.
